clar2wasm/tests/wasm-generation/main.rs

```diff
@@ -280,6 +282,10 @@ fn tuple(tuple_ty: TupleTypeSignature) -> impl Strategy<Value = Value> {
     })
 }
 
+fn block_range(upper_limit: u32) -> impl Strategy<Value = u32> {
+    1..=upper_limit
+}
```

Comment:
Should it be in *main.rs*? Will we reuse it? Why not directly using a range, where I could also decide what is the lower bound?

Reply:
Yep, doesn't make sense to have a function only for that. Removed.